app/design/frontend/UnitedArtsEducation/unitednow/Magento_Checkout/web/css/source/_extend.less

```diff
@@ -153,6 +153,25 @@
         }
     }
 
+    .cvv{
+        .field-tooltip{
+            right:initial;
+            left:64px;
+        }
+    }
+    //match normal colors and styles for sign in button in checkout.
+    .authentication-wrapper {
+        .action-auth-toggle{
+            font-weight:bold;
+            color:#575757;
+
+            &:hover{
+                color:#333333;
+                text-decoration:none;
+            }
+        }
+    }
```

Comment:
I don't think we need to modify the style of the sign-in link. (It matches the rest of the links on the site, so it should  be intuitive, and since it's not actually part of the nav bar in this case, I don't think it needs to match it.)